.github/workflows/ci.yml

```diff
@@ -11,23 +11,20 @@ jobs:
       matrix:
         include:
           - operating-system: 'ubuntu-20.04'
-            php-version: '7.2'
+            php-version: '7.4'
             job-description: 'with lowest deps'
             composer-flags: '--prefer-stable --prefer-lowest' # should be checked on lowest supported PHP version
 
           - operating-system: 'ubuntu-20.04'
-            php-version: '7.2'
+            php-version: '7.4'
             job-description: 'with Sf ^4'
             execute-flex-with-symfony-version: '^4' # Explicit check for Sf 4.x compatibility
 
           - operating-system: 'ubuntu-20.04'
-            php-version: '7.3'
+            php-version: '7.4'
             job-description: 'with Sf ^5'
             execute-flex-with-symfony-version: '^5' # Explicit check for Sf 5.x compatibility
 
-          - operating-system: 'ubuntu-20.04'
```

Comment:
Shouldn't we keep this one to test with Symfony 6.x?

Reply:
it's few lines below: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6203/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR30

Reply:
Yes but with PHP 8.0. Don't we want to test with latest dependencies on all supported PHP versions?

Reply:
idea here was always to test each supported Sf version at least once. we were not testing each PHP with Sf v6 - do you see big benefit of it, justifying the extra time CI would need ?

anyway, as I'm not changing the approach here, I would suggest to apply potential changes as separated PR

Reply:
I assumed using latest versions is the most common case so maybe we want to test it more intensively but I'm fine with current approach.

Reply:
so, we have "lowest PHP with lowest deps", "highest PHP with highest deps" and something between in the middle :D